inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp

```diff
@@ -393,6 +393,7 @@ ModelPtr FrontEnd::runCommonPasses(ie::ICNNNetwork& network, const UnsupportedLa
         }
 
         ie::NetPass::ConvertPrecision(*originalOrConvertNetwork, ie::Precision::I64, ie::Precision::I32);
+        ie::NetPass::ConvertPrecision(*originalOrConvertNetwork, ie::Precision::U32, ie::Precision::I32);
```

Comment:
Sync required logic with https://github.com/openvinotoolkit/openvino/pull/1304

Reply:
Look like these PRs can be merged separately...

Reply:
What I meant is are we sure that we do U32 -> I32 conversion in an expected way?